### PR TITLE
TASK-45118: Disabled purchase icon cases

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: perk-store-services/src/main/resources/locale/addon/PerkStore_en.properties [hunk 1]
```diff
@@ -232,6 +232,8 @@ exoplatform.perkstore.label.profile.members=Members
 
 exoplatform.perkstore.label.openInWallet=Open in wallet
 exoplatform.perkstore.label.addNewProduct=Add New Product
+exoplatform.perkstore.label.SoldOut=Sold Out
+exoplatform.perkstore.label.disabledProduct=Disabled Product
```
Comment (Rdidmohamedelhadi) [inline perk-store-services/src/main/resources/locale/addon/PerkStore_en.properties:236]: use the keys that already exist and change it in Crowdin

`exoplatform.perkstore.label.productFiltersDisabledProducts`
`exoplatform.perkstore.label.productFiltersSoldOutProducts`

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue [hunk 2]
```diff
@@ -196,13 +202,25 @@ export default {
       return this.cantBuyProduct ? '' : 'clickable ';
     },
     cantBuyProduct() {
-      return (this.disabledBuy || !this.walletEnabled || this.walletLoading) || !this.displayBuyButton;
+      return (this.disabledBuy || !this.walletEnabled || this.walletLoading) || !this.displayBuyButton || (this.product.price > this.balance);
```
Comment (Rdidmohamedelhadi) [inline perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue]: you are sure of this behavior ?
you have to check it with @mzorai

**Review by Rdidmohamedelhadi — CHANGES_REQUESTED** [review]

File: perk-store-services/src/main/resources/locale/addon/PerkStore_en.properties (hunk 1 above)
Comment (GouadriaHanen) [reply perk-store-services/src/main/resources/locale/addon/PerkStore_en.properties:236]: already used can't change them!

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue (hunk 2 above)
Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue]: checked and it's a new behavior!

**Review by Rdidmohamedelhadi — COMMENTED** [review]
Resolve conflicts please

**Comment by GouadriaHanen** [comment]
> Resolve conflicts please

done!

**Review by Rdidmohamedelhadi — APPROVED** [review]

**Pull request merged**